services/gateway-api/src/circuit_breaker/meta_breaker.py

```
@@ -0,0 +1,357 @@
+            }
+
+            try:
+                # Use asyncio-compatible approach
+                import aiohttp
+
+                async with aiohttp.ClientSession() as session:
+                    async with session.post(url, data=payload) as response:
+                        if response.status == 429:
+                            # Rate limit
+                            logger.warning("Meta rate limit hit")
+                            raise Exception("Rate limit exceeded")
```

Ruff (0.14.8)

Abstract `raise` to an inner function

(TRY301)